pkg/cluster/fixssh.go

```diff
@@ -49,11 +49,26 @@ func (m *manager) fixSSH(ctx context.Context) error {
 	}
 
 	for i := 0; i < 3; i++ {
-		nicName := fmt.Sprintf("%s-master%d-nic", infraID, i)
-
-		nic, err := m.interfaces.Get(ctx, resourceGroup, nicName, "")
+		// NIC names might be different if customer re-created master nodes
+		// see https://bugzilla.redhat.com/show_bug.cgi?id=188249 for more details
+		// installer naming  - <foo>-master{0,1,2}-nic
+		// machineAPI naming - <foo>-master-{0,1,2}-nic
+		nicNameInstaller := fmt.Sprintf("%s-master%d-nic", infraID, i)
+		nicNameMachineAPI := fmt.Sprintf("%s-master-%d-nic", infraID, i)
+
+		var iErr, err error
+		var nic mgmtnetwork.Interface
+		nicName := nicNameInstaller
+
+		nic, err = m.interfaces.Get(ctx, resourceGroup, nicName, "")
 		if err != nil {
-			return err
+			iErr = err // preserve original error
```

Comment:
I would prefer to instead nest the error so that the second one is not swallowed

Reply:
We dont nest errors in this code :/